bindings/py/performance/random_serialization_perf.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
I think you should put this in the Python namespace. Since I don't think we want to install it as a command-line tool, we can have it run via `python -m nupic.bindings.random_serialization_perf` or similar. There are two reasons for this:

1. It prevents us from cluttering the top-level of Python codebase with a "performance" directory
2. It makes sure that when you run the test you are running the installed version that matches the installed codebase. As it is, someone may run a checked out version of the test that differs from the installed version of nupic.bindings